app/models/mediaflux/http/query_request.rb

```diff
@@ -51,7 +51,7 @@ def build_http_request_body(name:)
               # TODO: there is a bug in mediaflux that does not allow the comented out line to paginate
               #      For the moment we will utilize the where clasue that does allow pagination
               # xml.collection collection if collection.present?
-              xml.where "asset in collection #{collection}" if collection.present?
+              xml.where "asset in collection or subcollection of #{collection}" if collection.present?
```

Comment:
I think this was meant to be `asset.query :collection #{collection} if collection.present?` but notice that this would in addition to whatever `where` clause we use.

Reply:
@hectorcorrea I'm just adding `or subcollection of` so that the updated code to get around the bug mimics what happens with adding `:collection`. The secondary where clause is still included.